src/SigasiProjectCreator/Creator.py

```diff
@@ -309,13 +309,18 @@ def write(self, destination):
             os.makedirs(settings_dir)
         prefs_file_path = "com.sigasi.hdt.{0}.{1}.prefs".format(self.lang, str(self.lang).title())
         prefs_file = os.path.join(settings_dir, prefs_file_path)
+        # TODO why not overwrite?
         if not os.path.exists(prefs_file):
```

Comment:
If we don't want to overwrite, we should check way earlier (at the options parsing) whether the destination directory exists.

Reply:
Overwrite or not needs an overhaul, but not in this ticket